Jellyfin.Plugin.Webhook/WebhookServerEntryPoint.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.Threading.Tasks;
+using Jellyfin.Plugin.Webhook.Configuration;
+using Jellyfin.Plugin.Webhook.Destinations.Mqtt;
+using MediaBrowser.Controller.Plugins;
+using MediaBrowser.Model.Plugins;
+
+namespace Jellyfin.Plugin.Webhook;
+
+/// <summary>
+/// WebhookServerEntryPoint.
+/// </summary>
+public class WebhookServerEntryPoint : IServerEntryPoint
+{
+    private readonly IMqttClients _mqttClients;
+
+    /// <summary>
+    /// Initializes a new instance of the <see cref="WebhookServerEntryPoint"/> class.
+    /// </summary>
+    /// <param name="mqttClients">Instance of the <see cref="IMqttClients"/> interface.</param>
+    public WebhookServerEntryPoint(IMqttClients mqttClients)
+    {
+        _mqttClients = mqttClients;
+        WebhookPlugin.Instance!.ConfigurationChanged += ConfigurationChanged;
```

Comment:
This should be subtracted in `Dispose`
I know Dispose is only called on server shutdown but might as well not leave the listener attached.